modules/account-lib/src/coin/cspr/utils.ts

```diff
@@ -285,11 +296,17 @@ export function getTransferId(transferTx: DeployUtil.ExecutableDeployItem): numb
 export function getDelegatorAddress(delegateTx: DeployUtil.ExecutableDeployItem): string {
   const delegatorAddress = delegateTx.getArgByName(DELEGATE_FROM_ADDRESS);
 
-  if (!delegatorAddress || !delegatorAddress.isString()) {
-    throw new InvalidTransactionError('Delegate / Undelegate does not have an address defined');
+  if (!delegatorAddress || !delegatorAddress.clType().tag) {
```

Comment:
should we still check for `string` type?

Reply:
Those checks are in the following lines